components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -366,8 +385,6 @@ class GleanTest {
 
     @Test
     fun `Don't schedule pings if there is no ping content`() {
-        EventsStorageEngine.clearAllStores()
-
         resetGlean(getContextWithMockedInfo())
```

Comment:
But this `resetGlean` is different than the one in `@Before`: that's due to the `getContextWithMockedInfo()`, which is required for the two specific tests to not crash :)

Reply:
Okay, I didn't realize that it would crash without the mocked version on that test.  What about the other `resetGlean` in the `Don't scheduled pings if metrics disabled` test?

Reply:
Good catch! I've further cleaned up this.